docs/architecture-decision-records/20230123-pipeline-config-files.md

```diff
@@ -0,0 +1,60 @@
+# Pipeline Config Files
+
+## Summary
+
+This document describes the behavior of provisioning pipelines using pipeline
```

Comment:
Really a nitpick: "config" has been around in our vocabulary for some time. I'd like to suggest using the complete word ("configuration") in "official" communication (docs, GitHub, web site, blog posts, etc.).